### ISSUE TITLE
Cypress CLI dry run option

### ISSUE THREAD

**Issue by rcasperson-jc** [cypress-io/cypress#24230]
### What would you like?

I would like to be able to see which tests Cypress will run based on tagging/spec without actually running them.

### Why is this needed?

I want this because there are cases where I need different setup based on which test cases will be run. This is also especially relevant for parallelizing test runs in CI. I may need to know all of the tests that will be run, so they can be split up for parallelization.

**Comment by WellingtonBraga** [comment]
Hi, do we have any news on this one?

**Comment by wviana** [comment]
Yes, I got into this because I wanted to list all tests for a a tag.
